calicoctl/commands/node/run.go

```diff
@@ -256,6 +274,58 @@ Description:
 		fmt.Println(err)
 		os.Exit(1)
 	}
+
+	// Create the command to follow the docker logs for the calico/node
+	logCmd := exec.Command("docker", "logs", "calico-node", "--follow")
+
+	// Get the stdout pipe
+	outPipe, err := logCmd.StdoutPipe()
+	if err != nil {
+		fmt.Println("Error querying calico/node logs:")
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
Yes - if we can't read Stdout then there is no point in us getting the logs since we won't be able to see them.